packages/connectivity/src/scp-cf/authorization-header.ts

```diff
@@ -133,6 +133,9 @@ interface AuthenticationHeaders {
   'Proxy-Authorization'?: string;
   'SAP-Connectivity-Authentication'?: string;
 }
+interface AuthenticationHeaderSAMLAssertion {
```

Comment:
[pp] Add a `\n` here ;) 
```suggestion

interface AuthenticationHeaderSAMLAssertion {
```